packages/workflow-core/src/lib/workflow-runner.ts

```diff
@@ -453,6 +472,9 @@ export class WorkflowRunner {
           UPDATE_CONTEXT: {
             actions: updateContext,
           },
+          DEEP_MERGE_CONTEXT: {
+            actions: deepMergeContext,
+          },
```

Comment:
We add 2 events to any workflow in the platform:
1. `UPDATE_CONTEXT` - replaces the context with the given context
2. `DEEP_MERGE_CONTEXT` - uses a deep merge functionality to update the context